.husky/pre-commit

```diff
@@ -1,4 +1,4 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-npm run format && npm run test
+npx lint-staged
```

Comment:
Adds formatted files to the commit, just a quality of life thing idk why I set this up without it 🤔